crates/ide/src/ide/completion.rs

```diff
@@ -164,6 +164,11 @@ impl Context<'_> {
             return None;
         }
 
+        // Do not complete inside comments.
+        if let SyntaxKind::COMMENT = self.token.kind() {
```

Comment:
There is already a token kind check just above this line. We can merge them into a single one.

Also can we add a test for this?